### ISSUE TITLE
Server leaking internal error details to client

### ISSUE THREAD

**Issue by dcolligan** [ga4gh/server#288]
Because we return exception.message to the client in the case where an exception is thrown that we haven't seen before, we can end up sending internal implementation details back to the client, which is both irrelevant to them and a potential security hazard.  For instance:

```
ga4gh.client: 500 {"errorCode": 3424984409, "message": "global name 'ReadGroupNotFoundException' is not defined"}
```

We should change this to only return the message when we know we have customized it, and return some generic "Internal Server Error" message otherwise.

**Comment by jeromekelleher** [comment]
OK good point; this will be easy enough to fix.

**Comment by dcolligan** [comment]
Fixed in #289
